File: seleccion.js
function selection(a) {

    for (let i = 0; i < a.length; i++) {

        let small = i;

        for (let j = i + 1; j < a.length; j++) {

            if (a[small] > a[j]) {
                small = j;
            }
        }

        // Intercambiar
        let temp = a[i];
        a[i] = a[small];
        a[small] = temp;
    }
}

function printArr(a) {

    for (let i = 0; i < a.length; i++) {
        process.stdout.write(a[i] + " ");
    }
}

let a = [65, 26, 13, 23, 12];

console.log("Arreglo antes de ser ordenado:");
printArr(a);

selection(a);

console.log("\nArreglo despues de ser ordenado:");
printArr(a);

File: seleccion.java
public class Main {

    public static void selection(int[] a) {

        for (int i = 0; i < a.length; i++) {

            int small = i;

            for (int j = i + 1; j < a.length; j++) {

                if (a[small] > a[j]) {
                    small = j;
                }
            }

            // Intercambiar
            int temp = a[i];
            a[i] = a[small];
            a[small] = temp;
        }
    }

    public static void printArr(int[] a) {

        for (int i = 0; i < a.length; i++) {
            System.out.print(a[i] + " ");
        }
    }

    public static void main(String[] args) {

        int[] a = {65, 26, 13, 23, 12};

        System.out.println("Arreglo antes de ser ordenado:");
        printArr(a);

        selection(a);

        System.out.println("\nArreglo despues de ser ordenado:");
        printArr(a);
    }
}